ansible_rulebook/cli.py

```diff
@@ -173,6 +173,12 @@ def get_parser() -> argparse.ArgumentParser:
         "Default is sequential, actions will be run only after the "
         "previous one ends",
     )
+    parser.add_argument(
+        "--hot-reload",
+        help="Should perform hot-reload on rulebook file changes",
```

Comment:
@tarilabs I am thinking this option is only available when the ansible-rulebook is running in non worker mode. Should we mention that in the help. Since it will be ignored when run in worker mode.

Reply:
> Should we mention that in the help

@mkanoor in the inline-help per this suggestion, or somewhere else?

```suggestion
        help="Will perform hot-reload on rulebook file changes (when running in non-worker mode). This option is ignored in worker mode.",
```